src/async_turtle.rs

```diff
@@ -71,7 +71,11 @@ impl AsyncTurtle {
         Self {client, id, angle_unit}
     }
 
-    pub async fn forward(&mut self, distance: Distance) {
+    pub async fn forward<T>(&mut self, distance: T)
+    where
+        f64: From<T>,
```

Comment:
This could also be written as:
```suggestion
        Distance: From<T>,
```